package.json

```diff
@@ -15,7 +15,14 @@
     "lint:all:markdownlint": "npm run lint:markdownlint -- \"**/*.md\" --ignore-path .gitignore",
     "lint:all:prettier": "npm run lint:prettier -- \"**/*.{ts,json,css,scss,vue,html,md}\" --ignore-path .gitignore",
     "lint:all:stylelint": "npm run lint:stylelint -- \"**/*.{vue,css,scss}\" --ignore-path .gitignore",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test:e2e": "start-server-and-test preview :5050 'cypress open'",
+    "test:e2e:ci": "start-server-and-test preview :5050 'cypress run'",
+    "cypress": "cypress open",
```

Comment:
Do we need this script? How does it get used in the testing workflow?

```
    "cypress": "cypress open",
```

Reply:
No, I was just testing why cypress was not opening. I forgot to remove it.